src/ServiceControl.Transports.ASBS/Helper.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace ServiceControl.Transports.ASBS
+{
+    using System;
+    using System.Security.Cryptography;
+    using System.Text;
+    using NServiceBus;
+
+    public static class Helper
+    {
+        public static void ConfigureTransport(this TransportExtensions<AzureServiceBusTransport> transport, TransportSettings transportSettings)
+        {
+            transport.ConfigureSanitization();
+
+            transport.Transactions(TransportTransactionMode.SendsAtomicWithReceive);
```

Comment:
This the default transaction mode, but I guess you're just specifying it regardless?

Reply:
We specify it in all transports, ergo here as well